source/index.html

```diff
@@ -26,32 +27,42 @@
   ></script>
 
 <!-- Call our scripts -->
-<script type="module" src="js/createRecipe.js"></script>
 <script type="module" src="js/main.js"> </script>
+<script type="module" src="js/createRecipe.js"></script>
 <script type="module" src="js/components/RecipeCard.js"> </script>
 <script type="module" src="js/Router.js"> </script>
 
-  <link rel="shortcut icon" href="..//favicon.ico" type="image/x-icon">
+	<link rel="shortcut icon" href="..//favicon.ico" type="image/x-icon">
+
+<!-- Site HTML: -->
   <title>Recipe Book</title>
 </head>
 <body>
-  <nav>
-    <div class="nav-logo">
-      <img src="./images/pot.svg" alt="" class="nav-icon">
-      <span class="nav-title">Recipe Book</span>
-    </div>
-    <!-- TODO: Call Router -->
-    <a href="./landing.html">Home</a>
-    <a href="./explore.html">Explore</a>
-    <a href="./saved-recipes.html">Saved Recipes</a>
-    <!-- TODO: Search bar should call Router and display search results, fetch results -->
-    <form action="./search-results.html" class="search-bar">
-      <input type="text" name="" id="" placeholder="Search recipes">
-      <button type="submit">
-        <i class="fa fa-search"></i>
-      </button>
-    </form>
-  </nav>
+	<!-- Header of website:-->
+	<section class="website-header">
+		<nav>
+			<div class="nav-logo">
+				<img src="/source/images/pot.svg" alt="" class="nav-icon">
+				<span class="nav-title">Recipe Book</span>
+			</div>
+			<!-- TODO: Call Router -->
+      <!-- Note: the onclick="return false;" here is required for SPA/router;-->
+      <!-- -otherwise, page will reload and will get 404 error for requested link -->
+      <!-- (If don't use this, then make them buttons to avoid reload.)-->
+			<a href="" onclick="return false;" id="Home_Link">Home</a>
```

Comment:
i think it's a little cleaner to just change all of these ` <a>` tags to `<button>` elements and define event handlers in main.js for them that use the router object so we don't need to modify anything else in the navbar